README.md

```diff
@@ -1 +1,46 @@
-# capstone-js
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Pokemon API
+
+> This is a webpage where you could find some of your favorite pokemons! Come and check it out!
+
+## Built With
+
+- Html
+- JavaScript
+- Bootstrap
+- Sass
+
+## Live Demo
+
+[Live Demo Link](https://yersel500.github.io/capstone-js/dist/
+)
+
+## Authors
+
+👤 **Esteban Munoz**
+
+- GitHub: [@estebanmual](https://github.com/estebanmual)
+- Twitter: [@estebanmual](https://twitter.com/estebanmual)
+- LinkedIn: [Esteban Munoz](https://linkedin.com/in/estebanmual)
+
+👤 **Yersel**
+
+- GitHub: [@yersel500](https://github.com/yersel500/)
+- Twitter: [@YerselHurtado](https://twitter.com/YerselHurtado)
+- LinkedIn: [Yersel Hurtado](https://www.linkedin.com/in/yersel-hurtado/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] (Optional) It would be nice to create an _**Acknowledgement Section**_ that mentions the Pokemon and Involvement API used for this project. 👍